src/gentropy/dataset/study_locus.py

```diff
@@ -139,6 +139,20 @@ class CredibleInterval(Enum):
     IS99 = "is99CredibleSet"
 
 
+class FinemappingMethod(Enum):
```

Comment:
This is quet good, but mayby using the `ColocalisationMethod.METHOD_NAME` would be enough, so we do not maintain two label sources?